tools/sourcerepo-sync/source_sync.py

```diff
@@ -0,0 +1,109 @@
+"""Utility to sync sourcerepos from local yaml to datastore."""
+from google.cloud import datastore
+import yaml
+import os
+import sys
+import argparse
+
+
+def main() -> None:
+  parser = argparse.ArgumentParser(
+      description="Sync source repositories from local yaml to datastore.")
+  parser.add_argument(
+      "--dry-run",
+      action=argparse.BooleanOptionalAction,
+      dest="dryrun",
+      default=True,
+      help="Abort before making changes")
+  parser.add_argument(
+      "--verbose",
+      action=argparse.BooleanOptionalAction,
+      dest="verbose",
+      default=False,
+      help="Display records being operated on")
+  parser.add_argument(
+      "--kind",
+      action="store",
+      dest="kind",
+      default="SourceRepositoryTest",  # Change to SourceRepository
+      help="The datastore kind to operate on")
+  parser.add_argument(
+      "--project",
+      action="store",
+      dest="project",
+      default="oss-vdb-test",
+      help="GCP project to operate on")
+  args = parser.parse_args()
+
+  if args.project == 'oss-vdb-test':
+    file = 'source_test.yaml'
+  elif args.project == 'oss-vdb':
+    file = 'source.yaml'
+  else:
+    print('Invalid project')
+    return
+  local_sourcerepos = []
+  with open(os.path.join(sys.path[-1] + '/', file), 'r') as f:
+    local_sourcerepos = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
Can you clarify/add a comment here as to why the Loader has to be manually specified?

Reply:
It's just the arguments yaml.load() requires - do you still want me to comment this?

Either way, this did make me realise that CLoader is a thing that exists that supposedly runs a lot faster than FullLoader, so I've swapped them out :).

Reply:
Hmm I think you can exclude the Loader argument and it'll default to the `Loader` (I can't figure out what FullLoader is, literally doesn't exist in the documentation :confused: ?? ). 

I think CLoader requires LibYAML library bindings, which we should avoid to simplify the build process.

Also can you see if you can use `safe_load` here as well?

Reply:
Yes! `safe_load` works